web/assets/js/repository.js

```diff
@@ -0,0 +1,22 @@
+$(document).ready(function() {
```

Comment:
should be in path `app/Resources/assets/js`.
Then gulp shoud be run to append the file to `web/assets/js/main.js`